geoportal-application/geoportal-harvester-war/src/main/webapp/hrv/style/main.css

```diff
@@ -36,104 +75,162 @@ html,body {
 }
 
 .h-nav-layout {
-  width: 120px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .h-nav {
-  height: 100%;
+  flex: 0 1 auto;
+  display: flex;
 }
 
 .h-nav > .h-nav-button {
+  margin-right: 16px;
+  padding: 12px 16px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #404040;
+  border-bottom: 4px solid #ffffff;
   cursor: pointer;
-  border: #ebebeb solid thin;
-  margin-bottom: 0.5em;
-  margin-right: 1em;;
-  padding-top: 4px;
-  padding-right: 6px;
-  padding-bottom: 4px;
-  padding-left: 6px;}
+}
 
 .h-nav > .h-nav-button:hover {
-  background-color: lightgray;
+  border-bottom: 4px solid #2890ce;
+}
+
+.active-tab {
+  border-bottom: 4px solid #2890ce !important;
+}
+
+.h-version {
+  flex: 1;
+  align-self: center;
+  text-align: right;
+  font-size: 14px;
+  line-height: 1;
+  color: #404040;
 }
 
 .h-stage-layout {
-  
+  display: flex;
+  flex: 1 1 auto;
+  background: #f3f3f3;
 }
 
 .h-stage {
-  height: 100%;
+  display: flex;
+  flex: 1 1 auto;
 }
 
 .h-brokers {
-  margin-bottom: 2em;
+  flex: 1 1 auto;
+  max-width: 960px;
+  margin: 0px auto;
+  padding: 12px;
+}
+
+.h-brokers-layout {
+  height: 100%;
+  display: flex;
+}
+
+.h-brokers-output {
+  flex: 1;
+}
+
+.h-brokers-input {
+  flex: 1;
+  margin-right: 80px;
 }
 
 .h-brokers-footer {
-  margin-top: 2em;
 }
 
 .h-brokers-content {
-  display: table;
+  display: flex;
 }
```

Comment:
Adding the following would fix the issue with the layout:
`flex-direction: column;`